src/bin/main.ts

```diff
@@ -17,7 +17,7 @@ const uploader = new HubspotUploader(console);
 
 const runLoopConfig = runLoopConfigFromENV();
 const notifier = SlackNotifier.fromENV(console);
-notifier?.notifyStarting();
+void notifier?.notifyStarting();
```

Comment:
This looks a bit redundant IMO. Is it an ESLint rule?

Reply:
@khanguyen88 Yeah ESLint recommended it as a fix to un-awaited promises without using await or then. At first I thought it was kind of weird but it's starting to grow on me as like an explicit recognition that it's an intentional but non-awaited-or-then'd async promise.